sw/device/demos/aes_hmac/aes_hmac_demo.c

```diff
@@ -0,0 +1,166 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/aes.h"
+#include "sw/device/lib/common.h"
+#include "sw/device/lib/uart.h"
+
+#define LENGTH_KEY 32
+#define MAX_LENGTH_TEXT 128
+
+/**
+ * Wait for AES unit to have valid output data and then
```

Comment:
this comment seems out of place with the functionality of this routine

Reply:
Done.